src/libraries/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/PosixSignalRegistrationTests.Unix.cs

```diff
@@ -57,7 +59,15 @@ public void SignalHandlerCalledForKnownSignals(PosixSignal s)
                     semaphore.Release();
                 });
 
-                kill(signal);
+                // Use 'kill' executable with signal name to validate the signal pal mapping.
+                using var process = Process.Start(new ProcessStartInfo
+                    {
+                        FileName = "kill",
+                        ArgumentList = { "-s", signalStr, Environment.ProcessId.ToString(CultureInfo.InvariantCulture) }
```

Comment:
Interesting. kill is expecting negative values? I always interpreted a command like "kill -9 12345" as sending signal number 9, denoted with a dash in front of it like other arguments,  to pid 12345. But is "-9" here actually a negative number expected by kill?

Reply:
`kill` accepts this for the signal: `kill  [-signal|-s signal]`. I've explicitly added the `-s`.
Here we're calling it with a name: `kill -s SIGINT _pid_`. I prefer to use the `-s` syntax with names, but `kill -SIGINT _pid_` also works.